playground/src/app.js

```diff
@@ -48,7 +48,7 @@ addSplitPaneBehavior(document.querySelector('main'));
 // configure error list behavior
 const errorIndicator = document.getElementById('error-indicator');
 const errorListContainer = document.getElementById('error-list');
-const errorList = ErrorList.createErrorList(errorListContainer, errorIndicator);
```

Comment:
This part is still needed as it creates the error list: `ErrorList.createErrorList(errorListContainer, errorIndicator);`. Only the variable is not used.

Reply:
thanks for your feedback @sebastianbenz , I've corrected this mistake. Sorry about that!